discretize/tree_mesh.py

```diff
@@ -135,31 +133,26 @@ class TreeMesh(_TreeMesh, BaseTensorMesh, InnerProducts, TreeMeshIO):
             "gridhEz": "hanging_edges_z",
         },
     }
+    _items = {'h', 'origin', 'cell_state'}
```

Comment:
are `h` and `origin` defined on the BaseTensorMesh? It likely doesn't make too much of a difference if we want to inherit those or not, just wanted to raise it to ask

Reply:
They are, but BaseTensorMesh also accepts the coordinate system and other kwarg parameters that are currently not supported on TreeMesh, so it was best to just restate the items here.

Reply:
:+1: makes sense. Thanks Joe!